pkgs/development/libraries/fontconfig/default.nix

```
@@ -15,21 +15,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fontconfig";
-  version = "2.13.93";
+  version = "2.13.94";
 
   src = fetchurl {
     url = "http://fontconfig.org/release/${pname}-${version}.tar.xz";
```

```suggestion
    url = "https://fontconfig.org/release/${pname}-${version}.tar.xz";
```

or the final redirect which would be ``https://www.freedesktop.org/software/fontconfig/release/``.